engine/NetworkProcessorNet.go

```diff
@@ -91,14 +91,6 @@ func Peers(fnode *FactomNode) {
 		if fnode.State.GetHighestCompletedBlk() < fnode.State.GetTrueLeaderHeight()-35 {
 			// Discard all commits, reveals, and acks <= the highest ack height we have seen.
 			switch amsg.Type() {
-			case constants.COMMIT_CHAIN_MSG:
```

Comment:
Seems like this change makes the other message types return true when they would have fallen out and returned false.

Reply:
you are right. i have reversed these changes with https://github.com/FactomProject/factomd/pull/647/commits/a36264176241558509c57401e3f05b44e615009c